make-cookbook.lisp

```diff
@@ -36,9 +38,10 @@
    "type.md"
    "sockets.md"
    "os.md"
+   "ffi.md"
    "process.md"
    "systems.md"
-   ;; "win32.md"
+   "win32.md"
```

Comment:
The win32 chapter was excluded from the html index and from the epub. It looks so old (done with LispWorks 4), it is Windows only, there are newer tools for that and newer cross-platform GUI tools… is it really relevant to include it back into the epub? Googlers can still find it online.

Reply:
Also the includes appear as is in the epub: `{% include code/w32-appendix-b.lisp %}`.